images/baremetal/manifests/0000_90_baremetal-installer_01_images.configmap.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: installer-images
+  namespace: openshift-config
```

Comment:
@patrickdillon I chose this namespace because it doesn't depend on any capabilities and is already used by the installer to create the install ConfigMap. But you know of a more appropriate one I'd be happy to change it. (I'd prefer not to create a whole new namespace.)